src/dashboard/store/modules/dashboard.js

```diff
@@ -9,6 +9,7 @@ export default {
   state: {
     creators: [],
     lastEdit: {},
+    metamaskStatusError: 'Please install MetaMask to be able to donate',
```

Comment:
Shouldn't this be null by default and then set to true if MetaMask isn't detected?

Reply:
No, when Error === null, everything is good in the world, so we set it to that first when everything is detected to be fine on line 49.